distribution/bwc/build.gradle

```diff
@@ -104,7 +104,7 @@ bwcVersions.forPreviousUnreleased { BwcVersions.UnreleasedVersionInfo unreleased
         dependsOn fetchLatest
         doLast {
             String refspec = System.getProperty("tests.bwc.refspec.${bwcBranch}", "${remote}/${bwcBranch}")
-            if (System.getProperty("tests.bwc.checkout.align") != null) {
+            if (System.getProperty("org.elasticsearch.bwc.checkout.align") != null) {
```

Comment:
There are other properties also here, like tests.bwc.refspec just above this too. Can we also just remove the tests prefix? For the refspec it is something we occationally need to specify locally, so having an extra long property name makes it both more cumbersome to type and difficult to remember.

Reply:
I'm less concerned about that one since it's only used in the exceptional case.

Reply:
We do also pass it from CI in the forward comparability tests.
+1 on having consistent parameter names for bwc

Reply:
I've renamed `tests.bwc.refspec` to `org.elasticsearch.bwc.refspec`. I've updated the associated infra PR as well.

Reply:
Can we please remove the o.e prefix? I type this whenever testing bwc changes, so the extra length does matter.

Reply:
Done. I've removed the verbose prefix and updated the corresponding infra PR.